shell/platform/fuchsia/flutter/tests/flatland_external_view_embedder_unittests.cc

```diff
@@ -461,6 +461,8 @@ TEST_F(FlatlandExternalViewEmbedderTest, SimpleScene) {
       IsFlutterGraph(parent_viewport_watcher, viewport_creation_token, view_ref,
                      /*layers*/
                      {IsImageLayer(frame_size)}));
+
+  EXPECT_EQ(fake_flatland().hit_regions().size(), 1u);
```

Comment:
Once you've added this to FakeTransform, you can just modify "IsFlutterGraph" above to incorporate checking for the hit regions

Reply:
Actually it might be "IsImageLayer" that you modify

Reply:
Done